shared/login/loading/index.tsx

```diff
@@ -33,8 +33,7 @@ const Splash = (props: Props) => {
     if (!__STORYBOOK__) {
       setShowFeedbackTrueLater()
     }
-    // eslint-disable-next-line
-  }, [])
+  }, [setShowFeedbackTrueLater])
```

Comment:
Is `Kb.useTimeout` going to return the same thing every render, or is this going to run on every render unless you add a `useCallback`?

Reply:
it already uses usecallback inside